create.go

```diff
@@ -58,6 +58,9 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 		if err := revisePidFile(context); err != nil {
 			return err
 		}
+		if err := reviseRootDir(context); err != nil {
+			return err
+		}
```

Comment:
This should go inside the `app.Before` function closure in `main.go` because `--root` is a global flag and we should sanitise it for all subcommands.